tests/export/test_xml.py

```diff
@@ -461,27 +461,29 @@ def get_xml(self):
         return xml
 
 
-class SeperateListsTestCase(TranslationTestCase):
+class SeperateListsIntoParentListTestCase(TranslationTestCase):
     expected_output = '''
         <ol class="pydocx-list-style-type-lowerLetter">
-            <li>AAA</li>
-        </ol>
-        <ol class="pydocx-list-style-type-decimal">
-            <li>BBB</li>
-        </ol>
-        <ol class="pydocx-list-style-type-lowerLetter">
-            <li>CCC</li>
+            <li>
+                AAA
+                <ol class="pydocx-list-style-type-decimal">
+                    <li>BBB</li>
+                    <li>CCC</li>
+                </ol>
+            </li>
+            <li>DDD</li>
```

Comment:
This testcase really helped me understand the new behavior and how it's superior to the previous behavior. Thanks for including it!